MAP/src/components/Legend/test/Legend.test.js

```diff
@@ -15,12 +15,15 @@ it('renders correctly when there are no items',  () => {
     expect(tree).toMatchSnapshot();
   });
 
-it('render Correctly with context data' , async () => {
+it('>>>render Correctly with context data' , async () => {
     const tree =  renderer.create(
         <AppContext.Provider value={envMock}>
```

Comment:
I'm thinking it might make sense for someone debugging this test in the future, if you edit the `envMock` right before creating the `const tree` so that when viewing the test code, it's easy to see what the expected content will be. for example:

```
let usedEnv = envMock
usedMock.components[1] = {
    "name":"Legend",
    "is_visible": true,
    "data": [
        {
            "title": "mapLegend.no",
            "worldStyle": "2"
        },
}
```

then you also don't need to keep changing the mock for each test case